template/Portfolio-Websites/React/v2/src/components/project/Project.js

```
@@ -0,0 +1,62 @@
+            <div
+              className="projdesc"
+              style={{ fontSize: '16px' }}
+            >
+              Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+              Temporibus at dolorum tempore sit labore voluptatibus similique cum
+              sapiente, officia accusantium.
+            </div>
+            <div className='projstackcont' style={{display: 'flex', justifyContent: 'space-between'}}>
+              <div
+                className="projstack"
+                dangerouslySetInnerHTML={{ __html: e.stack }}
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)